src/components/Planet/planet.css

```diff
@@ -17,11 +17,47 @@
   cursor: pointer;
 }
 
-.planet-icon:hover {
+.Mercury-img:hover {
+  transform: scale(.80);
+  transition: transform .5s ease-in-out;
+}
+
+.Venus-img:hover {
+  transform: scale(.90);
+  transition: transform .5s ease-in-out;
+}
+
+.Earth-img:hover {
+  transform: scale(.95);
+  transition: transform .5s ease-in-out;
+}
+
+.Mars-img:hover {
+  transform: scale(.85);
+  transition: transform .5s ease-in-out;
+}
+
+.Jupiter-img:hover {
+  transform: scale(1.20);
+  transition: transform .5s ease-in-out;
+}
+
+.Saturn-img:hover {
+  transform: scale(1.15);
+  transition: transform .5s ease-in-out;
+}
+
+.Uranus-img:hover {
+  transform: scale(1.10);
+  transition: transform .5s ease-in-out;
+}
+
+.Neptune-img:hover {
```

Comment:
Love that the effect is a different size depending on the size of the planet. Very cool idea!!